lasagne/layers/corrmm.py

```diff
@@ -8,21 +8,31 @@
 from .conv import conv_output_length, BaseConvLayer
 from ..utils import as_tuple
 
-from theano.sandbox.cuda.basic_ops import gpu_contiguous
-from theano.sandbox.cuda.blas import GpuCorrMM
-
+try:
+    from theano import gpuarray as gpu
+except ImportError:
+    from theano.sandbox import gpuarray as gpu
+enabled = gpu.pygpu_activated
+if not enabled:
+    try:
+        from theano.sandbox import cuda as gpu
+    except ImportError:
+        enabled = False
+    else:
+        enabled = gpu.cuda_enabled
+if enabled:
+    gpu_contiguous = gpu.basic_ops.gpu_contiguous
+    GpuCorrMM = gpu.blas.GpuCorrMM
+else:
+    raise ImportError(
+        "requires GPU support -- see http://lasagne.readthedocs.org/en/"
+        "latest/user/installation.html#gpu-support")  # pragma: no cover
```

Comment:
Can you please also include the comments I had in my suggestion? (Except for the one in square brackets.)